pkg/kubelet/pod_workers.go

```diff
@@ -580,6 +580,7 @@ func (p *podWorkers) UpdatePod(options UpdatePodOptions) {
 						syncedAt:           now,
 						startedTerminating: true,
 						finished:           true,
+						fullname:           kubecontainer.GetPodFullName(pod),
```

Comment:
For the specific scenes, do we need to add some tests?

Reply:
https://github.com/kubernetes/kubernetes/pull/107695#issuecomment-1019012431